pipeline/scripts/average_rating.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+sys.path.insert(0, "/workspace/butterfly/backend/utils/")
+
+
+def calculate_analytics(matches):
+    totals = dict()
+    count = dict()
+    averages = dict()
+    for match in matches:
+        totals[match["from_user"]] = match["value"] + totals.get(
+            match["from_user"], 0
+        )
+        count[match["from_user"]] = 1 + count.get(match["from_user"], 0)
+    keys = list(totals.keys())
+    for i in range(len(totals)):
+        averages[keys[i]] = totals[keys[i]] / count[keys[i]]
+    return averages
+
+
+def average_rating():
+    from firebase import get_db, initialize_firebase_for_api
```

Comment:
This import should go at the top of the file. I see now why you needed to add the `backend/utils` to the path.

Instead your imports should work like this:

```python
import sys

sys.path.append("./")

from pipeline.backend.firebase import get_db, initialize_firebase_for_api
```

It would be better to import from `pipeline` than `backend`, but I see that this meets your needs, so I can refactor it later.